src/matrix.jl

```diff
@@ -184,3 +184,12 @@ function ±(C::MT, S::MT) where {T, MT<:AbstractMatrix{T}}
 
     return IntervalMatrix(map((x, y) -> x ± y, C, S))
 end
+
+for op in (:Adjoint, :Bidiagonal, :Diagonal, :Hermitian,
+    :SymTridiagonal, :Symmetric, :Transpose, :Tridiagonal)
```

Comment:
```suggestion
           :SymTridiagonal, :Symmetric, :Transpose, :Tridiagonal)
```
I find that more readable.